r2/r2/lib/captcha.py

```diff
@@ -48,10 +48,14 @@ def getLayers(self, solution="blah"):
                  Distortions.SineWarp()))
 
 def get_iden():
-    return randomIdentifier(length=IDEN_LENGTH)
+    return self.randomIdentifier(length=IDEN_LENGTH)
```

Comment:
There isn't really a "self" to reference here. It's not done from within the class.